service/templates/projects.html

```diff
@@ -140,6 +140,17 @@ <h2> Settings for [[displayName]]</h2>
         <input class="form-control" type="text" ng-model="settingsFormData.display_name">
       </div>
     </div>
+    <div class="form-group">
+      <div class="col-lg-2 col-header">
+        <label class="control-label"> Identifier</label>
+      </div>
+      <div class="col-lg-4">
+        <input class="form-control" type="text" ng-model="settingsFormData.identifier">
+      </div>
+    </div>
+    <div class="muted small" style="padding-bottom:12px;width:60%;">
```

Comment:
Since this isn't a very common workflow, can you only show this message if someone types in the identifier field?